test/runtest.py

```diff
@@ -260,6 +260,47 @@ class ClassA(object):
             ray.put(obj)
 
 
+def test_serialize_dataclasses(ray_start):
+
+    from dataclasses import dataclass
+
+    # Serializing dataclass types
+
+    @dataclass
```

Comment:
This test only makes sense for Python 3.7, right? (which we currently don't include in our CI)